Cargo.toml

```
@@ -1,6 +1,8 @@
 [workspace]
-members = [
+default-members = [
     "packages/*",
+]
+members = [
     "tools/*",
```

```suggestion
members = [
    "packages/*",
    "tools/*",
```

According to the [docs of this key](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-default-members-field):

> When specified, default-members must expand to a subset of members.